apps/io-wallet-support-func/src/app/main.ts

```diff
@@ -34,6 +36,10 @@ const database = cosmosClient.database(config.azure.cosmos.dbName);
 
 const walletInstanceRepository = new CosmosDbWalletInstanceRepository(database);
 
+appInsights.setup().start();
```

Comment:
this must be executed before other instructions and imports since it auto-instruments other imported modules, see https://github.com/gunzip/tracing-test-app/blob/main/src/functions.ts#L1